pkg/snapshotter/snapshotter_csi.go

```diff
@@ -382,11 +397,11 @@ func (c *csiDriver) RestoreStatus(pvcName, namespace string) (RestoreInfo, error
 	}
 
 	switch {
-	case c.pvcWaitingForFirstConsumer(pvc):
-		restoreInfo.VolumeName = pvc.Spec.VolumeName
-		restoreInfo.Size = size
-		restoreInfo.Status = StatusReady
 	case c.pvcBindFinished(pvc):
+		err := c.deletejob(pvc, namespace)
```

Comment:
In the corresponding kdmp PR we have `cleanupJobBoundResources()` which is called part of DE CR cleanup.  What is the use for calling it here?
Can we add some comments also on the same?

Reply:
This is to handle the success case where the volume gets bound.
The cleanup in reconciler needs to delete the job if volume does not become bound.